### PR TITLE
Added SparkRunner

### PR DESCRIPTION
**Pull request by prashastk**

### CODE REVIEW

File: blurr/core/record.py [hunk 1]
```diff
@@ -20,6 +20,8 @@ def __getattr__(self, name):
         """
         When attributes are not found, None is returned
         """
+        if name.startswith('__') and name.endswith('__'):
+            raise AttributeError
```
Comment (prashastk) [inline blurr/core/record.py]: This is needed so that the record object can be 'pickled'

Comment (mserranom) [reply blurr/core/record.py]: no need to add a text argument?

File: blurr/runner/spark_runner.py [hunk 2]
```diff
@@ -0,0 +1,108 @@
+"""
+Usage:
+    spark_runner.py --raw_data=<files> --stream_dtc=<file> [--window_dtc=<file>] --output_file=<file>
```
Comment (mserranom) [inline blurr/runner/spark_runner.py]: we were using `-` instead of `-` for argument separators.

**Comment by mserranom** [comment]
what are the next steps? (tests, integrate with the cli, ...)

File: blurr/core/record.py (hunk 1 above)
Comment (prashastk) [reply blurr/core/record.py]: This gets eaten up internally by Python. Added an argument anyways.

File: blurr/runner/spark_runner.py (hunk 2 above)
Comment (prashastk) [reply blurr/runner/spark_runner.py]: Changed to -.

**Comment by prashastk** [comment]
@mserranom -Yep, those two are the next steps. Although, I did not add tests intentionally right now. Need to think a bit about adding spark as a dependency to the tests. @indradeep was suggesting that we put the runners in a separate repo. Lets discuss this in our stand-up.

**Pull request merged**